### DISCUSSION TITLE
Filter by a column value

### DISCUSSION THREAD

**Discussion by Sasaa22222** [microsoft-search/pnp-modern-search#1902] [Q&A]
I have created a document library with a column "validity date" (for example until xxx) and would like to filter by this column "validity date" in my Pnp modern search filter
can anyone help?

<img width="718" alt="Screenshot" src="https://user-images.githubusercontent.com/73344482/157019947-36221f0a-b923-49e4-b43d-a254fca665dc.png">

**Comment by wobba** [comment]
1. Find the crawled property which was created for your column (ows_&lt;internal name of column&gt;)
2. Map the crawled property to RefinableStringXX
3. Re-index content with the column
4. Enjoy using RefinableStringXX as a filter

I should probably blog about this, but cover it somewhat in my troubleshooting search book. Might be some other posts on this topic on the web if you look.

**Reply by Sasaa22222** [reply]
thanks a lot. Can i ask where can i do step 3.? 
Why doesn't the Combobox or Checkbox show up if i put the column as RefinableStringXX in it?

**Reply by wobba** [reply]
https://docs.microsoft.com/en-us/sharepoint/crawl-site-content and the UX shows when values are returned.

**Reply by Sasaa22222** [reply]
Thank you. I tried to configure only those with today's or future date in the query template:

RefinableString05>={Today}

what am i making wrong?

**Reply by wobba** [reply]
Impossible to know what you are doing wrong without looking at the data. There are quiet a few steps you can take to analyze what goes on - steps I covered in a book https://www.techmikael.com/2019/12/troubleshooting-search-in-sharepoint.html.
